README.md

```diff
@@ -1,57 +1,10 @@
-# Northcoders News API
+# Hugo's News API
 
-## Background
+## Initial instructions
+First of all, you must create two .env* files and populate them with the database names.
```

Comment:
Would be good to say explicitly that they should create a file called '.env.development' and '.env.test' even though you reference them below we have to be super clear